judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/EnvironmentAndSequenceTest.java

```diff
@@ -0,0 +1,445 @@
+        EnvironmentVariables environmentVariables = new EnvironmentVariables(Map.of(
+                "JUDO_ENV_XXX", "xxxx"
+        ));
+        try {
+            environmentVariables
+                    .execute(() -> {
+                        assertThrows(RuntimeException.class, () -> envVarBoolDao.create(EnvVarBool.builder().build()));
+                        assertThrows(RuntimeException.class, () -> envVarDateDao.create(EnvVarDate.builder().build()));
+                        assertThrows(RuntimeException.class, () -> envVarTimeDao.create(EnvVarTime.builder().build()));
+                        assertThrows(RuntimeException.class, () -> envVarTimestampDao.create(EnvVarTimestamp.builder().build()));
+                        assertThrows(RuntimeException.class, () -> envVarIntegerDao.create(EnvVarInteger.builder().build()));
+                        assertThrows(RuntimeException.class, () -> envVarLongDao.create(EnvVarLong.builder().build()));
+                    });
+        } finally {
+            environmentVariables.teardown();
+            EnvironmentVariableMocker.deinitMocked();
+        }
+    }
+
+    /**
+     * Verify the !getVariable(“SYSTEM“, “key”) function calls and their shortcuts.
+     *
+     * @prerequisites Nothing
+     *
+     * @type Behaviour
+     * 
+     * @others If the parts of the expression won't evaluate in the "same time". The result of this test case is divergent.
+     *  Sometimes OK, sometimes Failed. In this case the equality need to replace with an investigation of the subtraction's result.
+     *
+     * @scenario
+     *  . Parse (and/or build) the model.
+     *
+     *  . The result of the model parsing (and/or building) is successful.
+     *
+     *  . Create an EnvVars entity instance (evs1) with the default values.
+     *
+     *  . Check the values of the following fields of the new entity instance (evs1).
+     *      * f1 == true
+     *      * f2 == true
+     *      * f3 == true
+     *
+     *  . The test is passed if all steps have been completed with the specified results.
+     */
+    @Test
+    @TestCase("TC012")
+    @Requirement(reqs = {
+            "REQ-SYNT-001",
+            "REQ-SYNT-002",
+            "REQ-SYNT-003",
+            "REQ-SYNT-004",
+            "REQ-TYPE-001",
+            "REQ-TYPE-006",
+            "REQ-TYPE-007",
+            "REQ-TYPE-008",
+            "REQ-MDL-001",
+            "REQ-MDL-003",
+            "REQ-ENT-001",
+            "REQ-ENT-002",
+            "REQ-EXPR-001",
+            "REQ-EXPR-002",
+            "REQ-EXPR-006",
+            "REQ-EXPR-007",
+            "REQ-EXPR-009",
+            "REQ-EXPR-012"
+    })
+    void testVerifyTheGetVariableShortcuts() {
+        EnvVars envVars = envVarsDao.create(EnvVars.builder().build());
+
+        LocalDate localDate = envVars.getCd().orElseThrow();
+        LocalDate localDate1 = envVars.getCd1().orElseThrow();
+        if (localDate.isBefore(localDate1)) {
```

Comment:
This shouldn't be needed.

Reply:
There is a slight possibility that different current dates get executed on different days.